exercises/exercise8.js

```diff
@@ -18,14 +17,17 @@
 
 
  // Example cash-in-drawer array:
- // [['PENNY', 1.01], Penny is 1 cent
- // ['NICKEL', 2.05], Nickel is 5 cents
- // ['DIME', 3.10], Dime is 10 cents
- // ['QUARTER', 4.25], Quater is 25 cents
- // ['ONE', 90.00], One Dollar is 100 cents
- // ['FIVE', 55.00],
- // ['TEN', 20.00],
- // ['TWENTY', 60.00],
- // ['ONE HUNDRED', 100.00]]
+ // [['PAISA', 25], 0.5 rupees is 1 Paisa
```

Comment:
@abdulqadir-ml can we rephrase the statement to something like There are 50 Paisa in 1 Ruppee. Also should add Ruppee after the number below for clarification?